src/Core/Domain/Product/QueryResult/ProductForEditing.php

```diff
@@ -125,6 +132,7 @@ public function __construct(
         $this->productSeoOptions = $productSeoOptions;
         $this->associatedAttachmentIds = $associatedAttachmentIds;
         $this->stockInformation = $stockInformation;
+        $this->specificPrices = $specificPrices;
```

Comment:
Didn't we consider moving specific prices list to new tab?
I have already started PR (https://github.com/PrestaShop/PrestaShop/pull/21231/files) to query them (but I've lacked info so I paused it).
If you put them all here in productForEditing we will not have an ability to add pagination/filtering etc.

Reply:
> Didn't we consider moving specific prices list to a new tab & use standard grid component?

I think it was an idea, but not decided for now.

It is possible to build a Grid without pagination and filtering 😄